### DISCUSSION TITLE
Breadcrumbs have lost their spacing

### DISCUSSION THREAD

**Discussion by brendanpetty** [epicmaxco/vuestic-ui#3949] [General]
When developing in 1.7.x, breadcrumbs were looking nice like this:
![image](https://github.com/epicmaxco/vuestic-ui/assets/40818011/1e52371e-605c-4591-84b3-90d848cac978)

But on upgrade 1.8.x they now look like this:
![image](https://github.com/epicmaxco/vuestic-ui/assets/40818011/6b972a4d-c320-49be-8aeb-ee3eecb958e8)

I thought installing Tailwind might help, but it doesn't. It seems the --va-breadcrumbs-separator-padding css variable doesn't get applied anywhere. I don't mind custom spacing or not, but need to improve the look of how these come out now. What am I missing?

**Comment by m0ksem** [comment]
I have the same issue and i'm not sure why styles not loading within component...

**Reply by m0ksem** [reply]
Made an issue for it, it is a bug in 1.8.* build.
Track here:https://github.com/epicmaxco/vuestic-ui/issues/3955
